GliaWidgets/Component/Bubble/BubbleView.swift

```diff
@@ -44,6 +54,27 @@ class BubbleView: UIView {
         ).cgPath
     }
 
+    func showOnHoldView() {
+        guard
+            let style = style.onHoldOverlay,
+            let userImageView = userImageView
+        else { return }
+
+        let onHoldView = OnHoldOverlayView(
+            style: style,
+            imageSize: .init(width: 26, height: 26)
```

Comment:
does it make sense to put this size value inside of style?

Reply:
That could work.